### PR TITLE
DDP-4052 Add instagram feed to Osteo welcome page

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
Picking up on PR from old repo that was here: https://github.com/broadinstitute/ddp/pull/1225
**Note:**
Just noticed that the PR includes a bunch of changes that are not part of the changes I made directly to this branch. Investigating....

### CODE REVIEW

File: ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.scss [hunk 1]
```diff
@@ -373,18 +364,7 @@
...
         margin: -6rem auto 11.875rem auto;
     }
-}
-
-@media screen and (max-width: 350px) {
-    .twitter-feed {
-        width: 250px;
-        height: 300px;
-    }
 }
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.scss:370]: Changes in the `ddp-osteo/src/app/components/welcome/welcome.component.scss` look correct, please don't revert them.

File: ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html [hunk 2]
```diff
@@ -178,16 +178,10 @@ <h2 class="social-media-header__text" translate>
...
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
+                <div>
+                    <p class="big bold" translate>HomePage.SocialMediaSection.InstagramFeedTitle</p>
+                    <toolkit-instagram-feed-lightswitch-plugin [widgetId]="lightswitchWidgetId">
+                    </toolkit-instagram-feed-lightswitch-plugin>
                 </div>
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html:185]: Only these changes are correct.

Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html:185]: @fat23cat Currently working with @zyme to straighten out this branch. Something went awry during the transition to new repo...

**Comment by fat23cat** [comment]
@MocanaAtBroad, can we remove this branch?

**Pull request closed without merging**